vmm/src/config.rs

```diff
@@ -482,6 +489,8 @@ pub struct CpusConfig {
     pub max_phys_bits: u8,
     #[serde(default)]
     pub affinity: Option<Vec<CpuAffinity>>,
+    #[serde(default)]
+    pub features: CpuFeatures,
```

Comment:
Can you please change it to `pub features: Option<CpuFeatures>,`? I think this is more in sync with the rest of the configuration. Also, it explicitly means we don't have to provide anything from the HTTP API perspective.

Reply:
Okay. My personal preference is to avoid unwrapping options when everything off is a sensible default in this case but it does make the http call a bit lighter.

Reply:
Also unsure if this makes any difference but kvm_hyperv is kind of the precursor to this idea and it is being set as a bool not an option.

Reply:
I think you might be able to get away without using an `Option` provided we have can have a default implementation for serde to use in its absence and if the fields also have defaults.

Reply:
Seems safe as `CpuFeatures` derives `Default`, `Serialize` and `Deserialize`.